### PR TITLE
bluetooth: tester: cas: Added support for testing Common Audio Service

### PR DESCRIPTION
**Pull request by MSEEHenrik**
- Set Member Lock
- Get Member RSI

### CODE REVIEW

File: tests/bluetooth/tester/src/btp_bap.c
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_bap.c:1]: How does the changes to BAP here related to CAS support?

Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_bap.c:1]: CAS/CAP needs to be able to set available and supported contexts in PACS.

Comment (Thalley) [reply tests/bluetooth/tester/src/btp_bap.c:1]: I see. 

Please add that in the commit message, and potentially also move this change into a separate change, as it seems more related to BAP (PACS) than CAP (CAS).

Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_bap.c:1]: Sure.  I will add it to the commit message.

Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_bap.c:1]: Changed my mind.  Moved PACS changes to a separate PR.

**Review by Thalley — APPROVED** [review]

**Review by fredrikdanebjer — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Comment by jhedberg** [comment]
Is this fixing any bugs? Non-documentation PRs need a "Fixes ..." entry.

**Comment by MSEEHenrik** [comment]
No - this is new implementation for the auto-pts testing application.
Has no direct impact on zephyr.

**Pull request merged**